docs/howto/garbage-collection/standalone-gc.md

```diff
@@ -0,0 +1,131 @@
+| `logging.format`           | Logs output format                                                                                 | "text"                         | "text","json"                                           |
+| `logging.level`            | Logs level                                                                                         | "info"                         | "error","warn",info","debug","trace"                    |
+| `logging.output`           | Where to output the logs to                                                                        | "-"                            | "-" (stdout), "=" (stderr), or any string for file path |
+| `logging.file_max_size_mb` | Max file size for logs output (relevant only if `logging.output` is set to a file path)            | 102400 (100MiB)                | number                                                  |
+| `logging.files_keep`       | Number of files to keep for logs rotation (relevant only if `logging.output` is set to a file path | 100                            | number                                                  |
+| `cache_dir`                | Directory to use for caching data during run                                                       | ~/.lakefs-sgc/data             | string                                                  |
+| `aws.max_page_size`        | Max number of items per page when listing objects in AWS                                           | not set (AWS defaults to 1000) | number                                                  |
+| `objects_min_age`          | Tells the GC Ignore any object that is last modified within this time frame ("cutoff time")        | "6h"                           | duration                                                |
+| `lakefs.endpoint_url`      | The URL to the lakeFS installation - should end with `/api/v1`                                     | NOT SET                        | URL                                                     |
+| `lakefs.access_key_id`     | Access key to the lakeFS installation                                                              | NOT SET                        | string                                                  |
+| `lakefs.secret_access_key` | Secret access key to the lakeFS installation                                                       | NOT SET                        | string                                                  |
+
+
+These keys can be provided in the following ways:
+1. Config file: Create a YAML file with the keys, each `.` is a new nesting level. \
+   For example, `logging.level` will be:
+   ```yaml
+   logging:
+     level: <value> # info,debug...
+   ```
+   Then, pass it to the program using the `--config path/to/config.yaml` argument.
```

Comment:
There isn't a default location it expects for? that's ok, just to clarify

Reply:
There is, it's mentioned in the "Command line reference" section:
> #### Flags:
> - `-c, --config`: config file to use (default is $HOME/.lakefs-sgc.yaml)